README.md

```diff
@@ -98,11 +98,13 @@ Wechaty already held lots of talk and got a lot of blogs in the past 4 years, he
 
 import { WechatyBuilder } from 'wechaty'
 
-WechatyBuilder.build()  // get a Wechaty instance
+const wechaty = WechatyBuilder.build({}) // get a Wechaty instance
+wechaty
   .on('scan', (qrcode, status) => console.log(`Scan QR Code to login: ${status}\nhttps://wechaty.js.org/qrcode/${encodeURIComponent(qrcode)}`))
   .on('login',            user => console.log(`User ${user} logged in`))
   .on('message',       message => console.log(`Message: ${message}`))
   .start()
```

Comment:
I think the above line should be removed because you have added the below line?

Reply:
I re-copied the original and modified it, I forgot to remove this line, it has been corrected.